lib/network/struct.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // How many times should we try to connect
 const maxRetry = 10
-const waitRetry = 1 * time.Second
+const waitRetry = 100 * time.Millisecond
 const timeOut = 5 * time.Second
```

Comment:
Yep, else the creation of the network-communications of the tree were really slow.